gangplank/cosa/v1.go

```diff
@@ -14,11 +14,11 @@ type Amis struct {
 }
 
 type Artifact struct {
-	Path               string `json:"path"`
-	Sha256             string `json:"sha256"`
-	SizeInBytes        int    `json:"size,omitempty"`
-	UncompressedSha256 string `json:"uncompressed-sha256,omitempty"`
-	UncompressedSize   int    `json:"uncompressed-size,omitempty"`
+	Path               string  `json:"path"`
+	Sha256             string  `json:"sha256"`
+	SizeInBytes        float64 `json:"size,omitempty"`
```

Comment:
Ah yes, JSON and 64 bit numbers.  We're all still dealing with the tech debt from Brendan Eich deciding that signed/unsigned/integer/float were too hard for programmers scripting web pages and JavaScript should just have a "number".

But clearly we're not going to come anywhere close to [MAX_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) for our sizes so  LGTM.